### PR TITLE
Correct several grammar errors in callback documentation

### PR DESCRIPTION
**Pull request by FreddieCrew**
As the title says, this pull request corrects a lot of grammar mistakes found in the documentation of several callbacks.

### CODE REVIEW

File: docs/scripting/callbacks/OnPlayerWeaponShot.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
 ---
 title: OnPlayerWeaponShot
-description: This callback is called when a player fires a shot from a weapon.
+description: This callback is called when a player fires a weapon.
 tags: ["player"]
 ---
 
 ## Description
 
-This callback is called when a player fires a shot from a weapon. Only bullet weapons are supported. Only passenger drive-by is supported (not driver drive-by, and not sea sparrow / hunter shots).
```
Comment (coool9) [inline docs/scripting/callbacks/OnPlayerWeaponShot.md:9]: Are driver drive-by, sea sparrow and hunter shots now supported?

Comment (FreddieCrew) [reply docs/scripting/callbacks/OnPlayerWeaponShot.md:9]: I was going to add it as a warning, but I forgot. I'll add it pronto.

File: docs/scripting/callbacks/OnDialogResponse.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
-| playerid    | The ID of the player that responded to the dialog.                                                                      |
-| dialogid    | The ID of the dialog the player responded to, assigned in ShowPlayerDialog.                                             |
-| response    | 1 for left button and 0 for right button (if only one button shown, always 1)                                           |
-| listitem    | The ID of the list item selected by the player (starts at 0) (only if using a list style dialog, otherwise will be -1). |
-| inputtext[] | The text entered into the input box by the player or the selected list item text.                                       |
+| playerid    | The ID of the player who responded to the dialog.                                                                      |
+| dialogid    | The ID of the dialog the player responded to, as assigned in ShowPlayerDialog.                                             |
+| response    | 1 for the left button, and 0 for the right button (if only one button is shown, it will always 1)                                           |
```
Comment (coool9) [inline docs/scripting/callbacks/OnDialogResponse.md:15]: It will always **be** 1

File: docs/scripting/callbacks/OnRconCommand.md [hunk 3]
```diff
@@ -39,17 +39,19 @@ public OnRconCommand(cmd[])
...
+
+The /rcon prefix is not included in the cmd parameter when a player types a command. If you use the print function here, it will send a message to both the player who typed the command in-game and the server log.
 
-"/rcon " is not included in "cmd" when a player types a command. If you use the "print" function here, it will send a message to the player who typed the command in-game as well as the log. This callback is not called when the player is not logged in as RCON admin. When the player is not logged in as RCON admin and uses /rcon login, this callback will not be called and OnRconLoginAttempt is called instead. However, when the player is logged in as RCON admin, the use of this command will call this callback.
+This callback is not called if the player is not logged in as an RCON admin. When a player uses /rcon login to log in, this callback will not be called, instead, OnRconLoginAttempt is called. Once logged in as an RCON admin, any subsequent commands will trigger this callback.
 
 :::
 
-:::warning
+:::tip
```
Comment (coool9) [inline docs/scripting/callbacks/OnRconCommand.md:50]: I believe this is a 'warning' instead of a 'tip'.

**Review by coool9 — APPROVED** [review]

File: docs/scripting/callbacks/OnRconCommand.md (hunk 3 above)
Comment (FreddieCrew) [reply docs/scripting/callbacks/OnRconCommand.md]: I agree. Done

**Review by adib-yg — APPROVED** [review]
